src/utils/pgwire/src/pg_response.rs

```diff
@@ -84,6 +84,7 @@ pub enum StatementType {
     ORDER_BY,
     SET_VARIABLE,
     SHOW_VARIABLE,
+    SHOW_TRANSACTION,
```

Comment:
Looks like unused? Also FYI a lot of variants here are actually not pg-compatible #14007 so it seems does not matter much

Reply:
I'll remove this and use SHOW_VARIABLE for now. We can fix others later.